### PR TITLE
Update @media/resolution index.md

### PR DESCRIPTION
**Pull request by scheinercc**
### Description

Add examples for additional resolution unit, dppx, x and dpcm, similar to #31902

### Motivation

In #30393 we ended up changing https://developer.mozilla.org/en-US/docs/Web/CSS/resolution#syntax, but the original request was for https://developer.mozilla.org/en-US/docs/Web/CSS/@media/resolution, which this pull request now includes.

### CODE REVIEW

**Comment by estelle** [comment]
Instead of adding new values on there own, let's change two of the three existing dpi values to different resolution units.

**Comment by scheinercc** [comment]
@estelle I pushed a new commit. The wording might need to be rephrased :)

File: files/en-us/web/css/@media/resolution/index.md [hunk 1]
```diff
@@ -24,22 +24,28 @@ The `resolution` feature is specified as a {{cssxref("&lt;resolution&gt;")}} val
...
+/* Exact resolution with unit `dpi` */
 @media (resolution: 150dpi) {
   p {
     color: red;
   }
 }
 
-/* Minimum resolution */
-@media (min-resolution: 72dpi) {
+/* Minimum resolution unit `dppx`, `x` */
```
Comment (estelle) [inline files/en-us/web/css/@media/resolution/index.md:34]:
```suggestion
/* Minimum resolution synonym units: `dppx` and `x` */
```

**Review by estelle — APPROVED** [review]
Thank you! 🎉

**Pull request merged**